app/controllers/articles_controller.rb

```diff
@@ -3,11 +3,15 @@ class ArticlesController < ApplicationController
     before_action :check_journalist, only: [:create, :new]
     before_action :authenticate_user!, only: [:create, :new]
     def index
-        @articles = Article.all
+        @categories = Category.all
+            if params[:category].present?
```

Comment:
the if block needs to be pushed one to the left

Reply:
done